js/books.js

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Please avoid linter disabling. You can avoid `max-classes-per-file` by modulating your code into different files. For this, you may use one file per class, and then use something like `import` to access those classes in the main file. Check this article out for info on how to do this: https://antjanus.com/blog/thoughts-and-opinions/how-i-structure-my-javascript-file/